src/rdb/rdb_raft.c

```diff
@@ -282,7 +279,7 @@ rdb_raft_load_replicas(struct rdb *db, uint64_t index)
 
 	db->d_replicas = daos_rank_list_alloc(nreplicas);
 	if (db->d_replicas == NULL)
-		goto err;
+		D_GOTO(err, rc = -DER_NOMEM);
```

Comment:
We really need to decide whether we use D_GOTO or not.  Looks like this file mostly uses it but then line 289 doesn't

Reply:
I was told to switch to goto.